HIP/hip-0000-event-serialization.md

```diff
@@ -0,0 +1,227 @@
+| EventDescriptor Object                                                 |
++------------+---------+---------------------------------+---------------+
+| # of bytes | Type    | Description                     | Typical Value |
++------------+---------+---------------------------------+---------------+
+| 4 bytes    | Integer | Serialization version of object | 0x00000003    |
++------------+---------+---------------------------------+---------------+
+| H bytes    | Hash    | The hash of the event           |               |
++------------+---------+---------------------------------+---------------+
+| 12 bytes   | NodeId  | The node that created the event |               |
++------------+---------+---------------------------------+---------------+
+| 8 bytes    | Long    | The generation of the event     |               |
++------------+---------+---------------------------------+---------------+
+| 8 bytes    | Long    | The roster round of the event   |               |
++------------+---------+---------------------------------+---------------+
+```
+
+### BaseEventHashedData
+
+```
++--------------------------------------------------------------------------------------------------+
+| BaseEventHashedData Object                                                                       |
++------------+-----------------------+---------------------------------------------+---------------+
+| # of bytes | Type                  | Description                                 | Typical Value |
++------------+-----------------------+---------------------------------------------+---------------+
+| 4 bytes    | Integer               | Serialization version of object             | 0x00000004    |
++------------+-----------------------+---------------------------------------------+---------------+
+| V bytes    | SoftwareVersion       | The software version that created the event |               |
++------------+-----------------------+---------------------------------------------+---------------+
+| 12 bytes   | NodeId                | The node that created the event             |               |
++------------+-----------------------+---------------------------------------------+---------------+
+| S bytes    | EventDescriptor       | The descriptor of the event's self parent   |               |
++------------+-----------------------+---------------------------------------------+---------------+
+| L bytes    | EventDescriptor[]     | A list of descriptors for other parents     |               |
++------------+-----------------------+---------------------------------------------+---------------+
+| 8 bytes    | Long                  | The round the event was created             |               |
++------------+-----------------------+---------------------------------------------+---------------+
+| I bytes    | Instant               | The time the event was created              |               |
```

Comment:
Why isn't `Instant` a predetermined number of bytes?

Reply:
It can be.  I was being lazy.  Modified to 16 bytes (two longs)